### ISSUE TITLE
permission error when cloning repositories via docker run

### ISSUE THREAD

**Issue by erikaheidi** [distroless/git#13]
When trying to clone a new repository on my local system using this image (with a volume set up) I get permission issues.

This is the command I'm trying to run:

```shell
docker run -it -v ${pwd}:/home/nonroot --rm distroless.dev/git clone https://github.com/distroless/static.git
```

The response I get:

```
fatal: could not create work tree dir 'static': Permission denied
```
Maybe we need to update `apko.yml` to set up `/home/nonroot` for the `nonroot` user?

**Comment by erikaheidi** [comment]
Update: I tried a few different things on the `apko.yaml` file. Adding the `paths` definition didn't change anything, the directory is already created and with correct permissions.

Changing the entry point to `/bin/sh -l` I was able to check that the permissions are right and I can create new directories as `nonroot` and even run a `git clone` there. I don't know why it doesn't work via docker run 🤔
